config/initializers/devise.rb

```diff
@@ -0,0 +1,308 @@
+# frozen_string_literal: true
+
+require 'devise_remote_user'
+
+DeviseRemoteUser.configure do |config|
+  config.env_key = lambda do |env|
+    if env['REMOTE_USER']
+      env['REMOTE_USER']
+    elsif Rails.env.development? && ENV['REMOTE_USER']
+      ENV['REMOTE_USER']
+    end
+  end
+  config.auto_create = true
+  config.auto_update = false
+  # config.logout_url = "http://my.host/path.to.remote.logout"
+  # config.attribute_map = { email: 'mail' }
```

Comment:
Eh?  The point was to disable the line altogether, not to provide an example.  We have a separate ticket for actually wiring stuff up when I expect this might get retouched.

Reply:
sure.  and totally fine if it's getting changed in follow on work.  just figured it might be sensible to line it up with other similar uses in our other apps, in case a future maintainer interpreted the difference as meaningful.

searchworks apparently does actually use it:  https://github.com/sul-dlss/SearchWorks/blob/615435f4dd12be7c2cd5defe947c720cb7475440/config/initializers/devise.rb#L13

i guess my point is, if this is the default, but if we would change the value were we to actually use it, i'd propose either getting rid of the line entirely, or changing it to match the usage in our other apps.